src/core/Akka.Remote/Transport/TransportAdapters.cs

```diff
@@ -161,6 +161,9 @@ public string AugmentScheme(string originalScheme)
         /// <returns>TBD</returns>
         public Address AugmentScheme(Address address)
         {
+            // HACK to resolve https://github.com/akkadotnet/akka.net/pull/3764
+            if (address.Protocol.StartsWith(AddedSchemeIdentifier))
```

Comment:
This caused some failures in the Akka.Remote test suite, where we ended up with mal-formed protocols that looked like "akka.akka.test" - those failures appeared to be racy. I'm not sure why they happened yet, but wanted to keep a fix in-place while I go looking for them. Going to assess how feasible they are to track down right now.

Reply:
Turns out this issue was caused by me supplying the wrong addresses into my unit tests, not the fault of anything to do with Akka.Remote. Undid this hack.